padertorch/contrib/examples/speech_enhancement/simple_mask_estimator/train.py

```diff
@@ -1,24 +1,23 @@
 """
 Very simple training script for a mask estimator.
-Saves checkpoints and summaries to $STORAGE_ROOT/simple_mask_estimator
+Saves checkpoints and summaries to $STORAGE_ROOT/speech_enhancement/simple_mask_estimator_{id}
 may be called with:
-python -m padertorch.contrib.examples.mask_estimator.simple_train
+python -m padertorch.contrib.examples.speech_enhancement.simple_mask_estimator.train
 """
-import os
-from pathlib import Path
 
 import numpy as np
-import torch
-
 import paderbox as pb
 import padercontrib.database.keys as K
+import padertorch as pt
+import torch
+from sacred import Experiment
 from padercontrib.database import JsonAudioDatabase
-from padercontrib.database.iterator import AudioReader
 from padercontrib.database.chime import Chime3
-from pb_bss.extraction.mask_module import biased_binary_mask
-import padertorch as pt
+from padercontrib.database.iterator import AudioReader
 from padertorch.summary import mask_to_image, stft_to_image
+from pb_bss.extraction.mask_module import biased_binary_mask
 
+ex = Experiment('Simple Mask Estimator')
 
 class SimpleMaskEstimator(pt.Model):
```

Comment:
Move this into a separate file `model.py` to be consistent with the other examples?

Reply:
Okay, you already did this while I was reviewing

Reply:
done